### DISCUSSION TITLE
[FEATURE] Release translations of game names

### DISCUSSION THREAD

**Discussion by Clemintina** [HypixelDev/PublicAPI#672] [Ideas]
Hello,

My idea is pretty simple. To release translated versions of the `https://api.hypixel.net/resources/games` endpoint. Since this appears to be a static file, simple locale codes could be used such as

- `https://api.hypixel.net/resources/games/fr` for French, 
- `https://api.hypixel.net/resources/games/de` for German 

For languages where there are dialects present on Hypixel, For example: Brazilian Portuguese,

- `https://api.hypixel.net/resources/games/pt-br` or, 
- `https://api.hypixel.net/resources/games/ptBR` 

and so on. This would allow us to implement official names in multiple languages to make a more accessible UI for end users.

**Comment by ConnorLinfoot** [comment]
Game and mode names are something we usually avoid translating if I recall correctly, so this isn't something that should be necessary from 3rd party tools.
